students/darrell/wk4 - assignments/assignment.py

```diff
@@ -0,0 +1,22 @@
+import json_save.json_save.json_save_dec as js
+
+
+@js.json_save
+class MyClass:
+
+    x = js.Int()
+    y = js.Float()
+    lst = js.List()
+
+    def __init__(self, x, lst):
```

Comment:
when you instantiate this class it will override the initialization of `x` and `lst` objects so the json_save won't have any affect here.